### ISSUE TITLE
Catagories Float at Top

### ISSUE THREAD

**Issue by or-dodson** [burtonator/polar-bookshelf#1163]
I scrolled down and realized that the categories bar in the document and they instead disappear as you scroll down. Coud you make this so that this bar floats at the top and stays there whether the documents are scrolled down.
![Capture](https://user-images.githubusercontent.com/48970011/67961636-93e48e00-fbc1-11e9-82e3-2dfd8c8355b5.PNG)

**Comment by burtonator** [comment]
Agreed that this is annoying... I'll try to take another stab at fixing it...
